script/Ciel/Day 5/05_01_CIEL05_3E.txt

```diff
@@ -791,42 +850,51 @@ With a bestial howl, the girl runs through the white hell.
 [sha:9aba9334f48f7cfa4d1d92b86464949ded7165ab]{
 -- Page 629, Offset 12511.
 -- 体内にこもった熱、加速する魔力を火のように排出する。
-The warmth contained inside her body and the accelerating magical energy are frantically discharged.
+Like a raging fire, she releases the heat and accelerating magical energy that have been accumulating in her body.
 }
 [sha:5798cac03f716b7eb7c8ceb7c59e8d0efdfcea08]{
 -- Page 629, Offset 12512.
 -- 氷河を踏み砕き、不動の吸血鬼に肉薄する。
-Treading and smashing through the ice torrent, she closes in on the immobile vampire.
+Plowing her way across the glacier, she closes in on the immobile vampire.
 }
 [sha:08da609d74de698c1d7a8c35ab0227ea4eb3c467]{
 -- Page 629, Offset 12513.
 -- 「“女――――――！”」
+// 女!
 "Woman――――――!"
 }
 [sha:69215a759893b795c3aa4ba90b34d1c7915f2b55]{
 -- Page 630, Offset 12514.
 -- 響き渡る<剣|てつ>の音。
-The sound of steel resonates.
+// SFX here sound mostly non-contact
+The sound of steel humming through the air.
 }
 [sha:b4ea4835e3574f629b3c1ec1ca46bfcbdb4d5137]{
 -- Page 630, Offset 12515.
 -- 少女が手にした物はあまりにも無骨な裁断機。
-Within her hand, the girl holds a very crude tool for cutting.
+In her hand, the girl holds a barbaric cleaver.
 }
 [sha:df9a4646d05b6a4a4c74323021ac27f3e547fe88]{
 -- Page 630, Offset 12516.
 -- ギロチンの刃を重ねた<百足|むかで>の如き<蛇腹|じゃばら><剣|けん>。
-A serpentine sword shaped like the trunk of a centipede, as though guillotine blades were stacked on top of each other.
+// 蛇腹剣 seems to be called a 'whip sword' in EN, a 'serpentine sword' seems to
+// conjure images of wavy blades instead of the telescoping / whipping thing.
+// However, later on in D5 there's some 文字通り business on 蛇, so I've kept
+// serpentine for 蛇腹 but in this instance called it a whip sword to establish
+// the connection in EN.
+A serpentine whip sword, like a centipede composed of stacked guillotine blades.
```

Comment:
whip blade maybe? "whip sword" sounds a little ridiculous

Reply:
I agree that it's dumb, but I think whip sword kinda encodes it better.